com.unity.formats.fbx/Editor/Sources/Recorders/FbxRecorder/FbxRecorderSettings.cs

```diff
@@ -1,4 +1,4 @@
-﻿#if ENABLE_FBX_RECORDER
+#if ENABLE_FBX_RECORDER
```

Comment:
Did anything actually change in this file?

Reply:
I think the editor config made some small whitespace changes